### DISCUSSION TITLE
Code signing with cert and sigh

### DISCUSSION THREAD

**Discussion by kigh143** [fastlane/fastlane#16925] [Q&A]
Hello, I am fairly new to Fastlane, I am using cert and sigh to code sign my application because I don't want to revoke my existing certificates. What is the best way to approach code signing using cert and sigh? 

**What I have tried**

1. I added these two lines in my FastFile
 ```
   get_certificates           # invokes cert
   get_provisioning_profile   # invokes sigh
 ```
6. I disabled automatic code signing 

**Error and getting **
`error: AppName has conflicting provisioning settings. AppName is automatically signed for development, but a conflicting code signing identity iPhone Distribution has been manually specified. Set the code signing identity value to "iPhone Developer" in the build settings editor, or switch to manual signing in the Signing & Capabilities editor. (in target 'AppName' from project 'AppName').`

Thank you.

**Comment by DeathStalker403** [comment]
You need to disable automatic code signing just include it at first line of lane

https://docs.fastlane.tools/actions/automatic_code_signing/

**Comment by techinpark** [comment]
why didn't you use `match`?

I think is very simple solution.  check out here. 
https://docs.fastlane.tools/actions/match/

**Comment by Pietro-Putelli** [comment]
Match does not work, it continually say me that it's impossible to access the git repository, even if the token is set properly.
